docs/tutorials/compaction-cow-mor.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Compaction behavior - Copy on Write(COW) vs Merge on Read(MOR)
+description: This documentation provides details on iceberg table compaction behavior for Copy-on-Write (COW) and Merge-on-Read (MOR) tables.
+last_update:
+  date: 06/11/2024
+  author: Alokh Pullanikkatt
+---
+
+import Img from '@site/src/components/Img';
+
+## Introduction
+In today’s data-driven landscape, efficient data management is essential for organizations leveraging large-scale analytics. This document focuses on the behavior of table compaction operations for Copy-on-Write (COW) and Merge-on-Read (MOR) tables.
+
+Table compaction plays a vital role in optimizing performance by consolidating smaller data files into larger ones, which reduces overhead and enhances query efficiency. This guide will walk you through the setup of catalogs and tables, data loading, updates, and the analysis of snapshots, providing a clear understanding of the compaction process within Iceberg and its benefits for effective data management.
+
+## Setup
+### Create a catalog
+
+<Img src="/img/tutorial/compaction-cow-mor/create-catalog.png" alt="Create catalog"/>
+
+:::info To ensure the newly created catalog is detected, you may need to restart the lakehouse if it was already running.
+:::
+
+### Create database and tables
+```
```

Comment:
Plz try to add `sql showLineNumbers` after ```

Reply:
Done